### ISSUE TITLE
Item Icon

### ISSUE THREAD

**Issue by oohLexa** [Kekilla0/Item-Macro#40]
Hi everyone,

I'm using simple worldbuilding, with the new update I'm now able to drag anything from the items list of an actor and drop it to roll a macro, but the icon of the item reset, like it wont stay, it's only the default dice icon.

Is there a way to keep the item icon that you drag in the macro bar? Thanks in advance.

**Comment by iwontgiveyoumyname** [comment]
Yeah, so everything works pertectly, but each time I drag and drop the item from the item list in the macro bar, the icon of the items dispear and instead it's the dice-target.svg icon, I don't know why
